pkg/models/querybuilder_performer.go

```diff
@@ -158,6 +158,45 @@ func (qb *PerformerQueryBuilder) Query(performerFilter *PerformerFilterType, fin
 		query.addArg(gender.Value.String())
 	}
 
+	if isMissingFilter := performerFilter.IsMissing; isMissingFilter != nil && *isMissingFilter != "" {
+		switch *isMissingFilter {
+		case "image":
+			query.addWhere("performers.image IS NULL")
```

Comment:
Image is never null because it is required to be not null in the schema.

Reply:
Is there an easy way of identifying the default images directly in the database?

Reply:
Removed for now.